Italian Translations Setup

pcerqua-ipzs opened the pull request:
## Title

Italian Translations Setup

## Content

- Adds translations from https://github.com/italia/eid-wallet-it-docs/pull/661;
- Standardizes script configurations between English and Italian documentation;
- Contains bidirectional alignments of RST best practices as specified in `CONTRIBUTING-RULES.md`.
- Addresses https://github.com/italia/eid-wallet-it-docs/issues/651

pcerqua-ipzs committed:
feat: integrate translations from PR #661 and align EN/IT documentation scripts & configurations

pcerqua-ipzs committed:
fix: added newline at end of files

pcerqua-ipzs committed:
fix: added newline at end of file

pcerqua-ipzs committed:
fix: fixed some "undefined label" warnings

pcerqua-ipzs committed:
fix: conf.py script enhancements to work on paths containing spaces and to launch plantuml in headless mode

peppelinux commented on docs/it/brand-identity.rst:
```diff
@@ -0,0 +1,136 @@
+Naming 
+------
+
+“Sistema IT-Wallet" (IT-Wallet System in inglese) o “IT-Wallet" in forma contratta è il nome ufficiale che DEVE essere utilizzato in contesti scritti o verbali, fisici o digitali. Di seguito i relativi requisiti per la corretta scrittura e pronuncia: 
```
```suggestion
"Sistema IT-Wallet" (IT-Wallet System in inglese) o "IT-Wallet" in forma contratta è il nome ufficiale che DEVE essere utilizzato in contesti scritti o verbali, fisici o digitali. Di seguito i relativi requisiti per la corretta scrittura e pronuncia: 
```

pcerqua-ipzs committed:
Update docs/it/brand-identity.rst

Co-authored-by: Giuseppe De Marco <demarcog83@gmail.com>

pcerqua-ipzs committed:
fix: gitguardian configuration fixes

pcerqua-ipzs committed:
fix: gitguardian configuration fix with exact paths

pcerqua-ipzs committed:
Merge branch 'italia:versione-corrente' into italian-translation-setup

pcerqua-ipzs committed:
fix: introduced fixed PID section for both english and italian

peppelinux commented on docs/it/brand-identity.rst:
```diff
@@ -0,0 +1,136 @@
+Brand Identity del Sistema IT-Wallet 
+====================================
+
+La Brand Identity del Sistema IT-Wallet definisce la personalità del sistema e si declina in una serie di elementi codificati che la contraddistinguono, tra cui il naming e gli elementi propri dell’Identità Visiva. Questa contribuisce a rafforzare e promuovere il Modello della Fiducia che sta alla base dell’intero ecosistema ovvero il modello che legittima la partecipazione degli attori al Sistem ...
```
```suggestion
La Brand Identity del Sistema IT-Wallet definisce il naming e gli elementi propri della Identità Visiva. Questa consolida l'implementazione del Modello della Fiducia. 
```

peppelinux replied:
non vogliamo ridefinire cos'è o cosa fa il modello della fiducia

vogliamo ridurre le parole

ogni volta che usiamo "ovvero" significa che non siamo stati chiari nel periodo precedente

steps88 replied:
La brand identity non definisce solo il naming e gli elementi dell'identità visiva, quelli sono solo esempi. D'accordo sulla 2 frase.
```suggestion
La Brand Identity del Sistema IT-Wallet definisce la personalità del sistema e si declina in una serie di elementi codificati che la contraddistinguono, tra cui il naming e gli elementi propri dell’Identità Visiva. Questa consolida l'implementazione del Modello della Fiducia.
```

peppelinux commented on docs/it/brand-identity.rst:
```diff
@@ -0,0 +1,136 @@
+Identità Visiva
+---------------
+
+Il Sistema IT-Wallet ha una sua propria un’Identità Visiva. Pertanto, i diversi attori dell’ecosistema DEVONO garantire che la propria Identità Visiva e quella delle proprie Soluzione Tecniche siano in grado di distinguersi e allo stesso tempo di dialogare e coesistere con quella del Sistema IT-Wallet.  
```
```suggestion
Il Sistema IT-Wallet ha una propria Identità Visiva. I partecipanti dell’ecosistema DEVONO implementarla.  
```

non vogliamo obbligare gli attori a poterci garantire qualcosa, ma obbligarli a fare qualcosa, come implementare chiaramente qualcosa.

steps88 replied:
il concetto è più complesso: dobbiamo mantenere il vincolo che l'identità visiva del sistema non deve essere applicata come identità visiva di una Soluzione tecnica o di un WP.

```suggestion
Il Sistema IT-Wallet ha una propria Identità Visiva. I partecipanti dell’ecosistema DEVONO applicarla in modo che si distingua e allo stesso tempo dialoghi e coesista con l'Identità Visiva propria e delle proprie Soluzioni Tecniche.  
```

peppelinux commented on docs/it/brand-identity.rst:
```diff
@@ -0,0 +1,136 @@
+
+Il Sistema IT-Wallet ha una sua propria un’Identità Visiva. Pertanto, i diversi attori dell’ecosistema DEVONO garantire che la propria Identità Visiva e quella delle proprie Soluzione Tecniche siano in grado di distinguersi e allo stesso tempo di dialogare e coesistere con quella del Sistema IT-Wallet.  
+ 
+In particolare, tutti gli Attori Primari POSSONO utilizzare le Risorse Ufficiali relative all’Identità Visiva del Sistema IT-Wallet. Il loro utilizzo DEVE avere come unico scopo quello di rappresentare la partecipazione al Sistema IT-Wallet e non di sostituire dell’Identità Visiva della propria Soluzione Tecnica. 
```
```suggestion
In particolare, tutti gli Attori Primari POSSONO utilizzare le Risorse Ufficiali relative all’Identità Visiva del Sistema IT-Wallet. Il loro utilizzo ha lo scopo di rappresentare la conformità al Sistema IT-Wallet senza sostituire l’Identità Visiva della propria Soluzione Tecnica. 
```

non è chiaro che effetti possa avere questo "POSSONO"

steps88 replied:
```suggestion
In particolare, tutti gli Attori Primari DEVONO utilizzare le Risorse Ufficiali relative all’Identità Visiva del Sistema IT-Wallet. Il loro utilizzo ha lo scopo di rappresentare la partecipazione al Sistema IT-Wallet senza sostituire l’Identità Visiva della propria Soluzione Tecnica. 
```

peppelinux commented on docs/it/brand-identity.rst:
```diff
@@ -0,0 +1,136 @@
+Logo 
+^^^^
+
+Il Logo è l'elemento grafico ufficiale che permette l’immediata riconoscibilità del Sistema IT-Wallet e ne promuove l’affidabilità. 
```
```suggestion
Il Logo è l'elemento grafico ufficiale che permette l’immediata riconoscibilità del Sistema IT-Wallet. 
```

steps88 replied:
Cosa non ti torna de "ne promuove l’affidabilità"?

peppelinux replied:
la valutazione dell'affidabilità non credo che avvenga nel dominio grafico, nel senso che un lupo può travestirsi da pecora. Io rimarrei nella riconoscibilità, mentre l'affidabilità deve essere valutata secondo quanto definito nell'apposita sezione relativa alla valutazione della fiducia.

peppelinux commented on docs/it/brand-identity.rst:
```diff
@@ -0,0 +1,136 @@
+
+- il Logo DEVE essere quello presente all’interno delle Risorse Ufficiali e DEVE seguire le relative specifiche di utilizzo disponibili nelle Risorse Ufficiali; 
+
+- il Logo DEVE essere utilizzato in formato ``application/svg +xml``; 
```
```suggestion
- il Logo DEVE essere utilizzato in formato ``application/svg+xml``; 
```

peppelinux commented on docs/it/brand-identity.rst:
```diff
@@ -0,0 +1,136 @@
+Componenti 
+^^^^^^^^^^
+
+I componenti rappresentano quegli elementi del Sistema IT-Wallet che abilitano l’Utente a interagire con le diverse Soluzioni Tecniche tramite la propria Istanza IT-Wallet.  
```
```suggestion
Sono definiti componenti gli elementi del Sistema IT-Wallet che abilitano l’Utente a interagire con le diverse Soluzioni Tecniche tramite la propria Istanza IT-Wallet.  
```

peppelinux commented on docs/it/brand-identity.rst:
```diff
@@ -0,0 +1,136 @@
+
+- gli Attori Primari POSSONO scegliere quale configurazione, tra quelle rese disponibili, implementare, ma DEVONO comunque garantire il corretto utilizzo dei componenti atomici come i Pulsanti di Ingaggio; 
+
+- gli Attori Primari DEVONO garantire il costante aggiornamento dei componenti, in linea con l’ultima versione resa disponibile. 
```
```suggestion
- gli Attori Primari DEVONO garantire il costante aggiornamento dei componenti, in linea con l’ultima versione resa disponibile. 
```

qui bisogna dire in che tempi

non è possibile pensare che all'immediata emissione di una nuova linea guida, tutti possano adeguarsi immediatamente. Bisogna normare dei tempi.

peppelinux replied:
idealmente, un adeguamento tecnico di media complessità può richiedere fino a 90 giorni

peppelinux replied:
https://github.com/italia/eid-wallet-it-docs/issues/728

peppelinux commented on docs/it/functionalities.rst:
```diff
@@ -0,0 +1,543 @@
+
+L'attivazione è il passaggio necessario per abilitare l'Utente all'utilizzo delle funzionalità della Soluzione IT-Wallet per l’ottenimento, la presentazione e la gestione dei propri Attestati Elettronici in modo sicuro. Il processo di attivazione consiste in un’operazione di Autenticazione dell'Utente sull’Istanza IT-Wallet tramite la propria identità digitale che consente la generazione del PID. ...
+
+Di seguito i requisiti funzionali e di Esperienza Utente che il Wallet Provider DEVE garantire attraverso la propria Soluzione IT-Wallet: 
```
```suggestion
Di seguito i momenti caratterizzanti l'Esperienza Utente nell'utilizzo della Soluzione IT-Wallet: 
```

Pare che di seguito non si leggano requisiti funzionali, ma una user journey

steps88 replied:
È importante che sia dato il requisito normativo al WP di realizzare il prodotto in modo che l'utente possa fare le azioni descritte

```suggestion
Di seguito i requisiti di Esperienza Utente che il Wallet Provider DEVE garantire attraverso la propria Soluzione IT-Wallet: 
```
Se sei d'accordo, aggiorniamo nel resto del documento.

peppelinux commented on docs/it/functionalities.rst:
```diff
@@ -0,0 +1,543 @@
+
+- **dal Catalogo dell’Istanza IT-Wallet**: l’Utente esplora l’elenco degli Attestati Elettronici di Attributi messi a disposizione dalla Soluzione IT-Wallet, seleziona quello di suo interesse e avvia la procedura di richiesta che si conclude con l’ottenimento dell’Attestato Elettronico di Attributi nell’Istanza IT-Wallet; 
+
+- **da un Touchpoint del Titolare di Fonte Autentica** (o del Fornitore di Attestati Elettronici di Attributi qualora coincida con il Titolare di Fonte Autentica): l’Utente interagisce col servizio digitale del Titolare di Fonte Autentica che gli permette di ottenere nella sua Istanza IT-Wallet uno specifico Attestato Elettronico di Attributi tramite un Pulsante di Ingaggio.  
```
```suggestion
- **da un Touchpoint del Titolare di Fonte Autentica** (o del Fornitore di Attestati Elettronici di Attributi qualora coincida con il Titolare di Fonte Autentica): l’Utente interagisce col servizio digitale del Titolare di Fonte Autentica che gli permette di ottenere nella sua Istanza IT-Wallet uno specifico Attestato Elettronico di Attributi tramite un Pulsante di Ingaggio (vedi :ref:`Credential Offer Endpoint <credential-issuance-endpoint.rst>`).  
```

qui mettere un riferimento al flusso di credential offer, definito nella sezione pid/eaa issuance

steps88 replied:
Reputi utile inserire questo riferimento specifico oppure è meglio mettere il riferimento a fondo sezione?

peppelinux commented on docs/it/functionalities.rst:
```diff
@@ -0,0 +1,543 @@
+
+- **da un Touchpoint del Titolare di Fonte Autentica** (o del Fornitore di Attestati Elettronici di Attributi qualora coincida con il Titolare di Fonte Autentica): l’Utente interagisce col servizio digitale del Titolare di Fonte Autentica che gli permette di ottenere nella sua Istanza IT-Wallet uno specifico Attestato Elettronico di Attributi tramite un Pulsante di Ingaggio.  
+
+Nonostante le modalità di avvio della richiesta siano diverse, i flussi di ottenimento condividono una struttura e un processo simili. Di seguito, sono illustrati i requisiti funzionali a supporto dell’Esperienza Utente del flusso di ottenimento di un Attestato Elettronico di Attributi dal Catalogo che il Fornitore di Soluzione IT-Wallet DEVE garantire attraverso la propria Soluzione IT-Wallet: 
```
```suggestion
Nonostante le modalità di avvio della richiesta siano diverse, i flussi di ottenimento condividono una struttura e un processo simili. Di seguito, sono illustrati i momenti caratterizzanti l’Esperienza Utente durante la fase di ottenimento di un Attestato Elettronico di Attributi dal Catalogo: 
```

steps88 replied:
```suggestion
Nonostante le modalità di avvio della richiesta siano diverse, i flussi di ottenimento condividono una struttura e un processo simili. Di seguito, sono illustrati i requisiti dell’Esperienza Utente del flusso di ottenimento di un Attestato Elettronico di Attributi dal Catalogo che il Fornitore di Soluzione IT-Wallet DEVE garantire attraverso la propria Soluzione IT-Wallet: 
```

peppelinux commented on docs/it/functionalities.rst:
```diff
@@ -0,0 +1,543 @@
+
+- **Modalità non supervisionata**: l’Utente, tramite l’Istanza IT-Wallet, presenta il PID e/o un set di Attributi contenuti in uno o più Attestati Elettronici a un dispositivo preposto (e.g. tornello, ATM, etc.). 
+
+Di seguito i requisiti funzionali a supporto dell’Esperienza Utente relativi a entrambe le modalità che il Fornitore di Soluzione IT-Wallet DEVE garantire attraverso la propria Soluzione IT-Wallet. 
```
```suggestion
Di seguito i momenti caratterizzanti l’Esperienza Utente relativi a entrambe le modalità di presentazione utilizzando una Soluzione IT-Wallet. 
```

steps88 replied:
```suggestion
Di seguito i requisiti dell’Esperienza Utente relativi a entrambe le modalità che il Fornitore di Soluzione IT-Wallet DEVE garantire attraverso la propria Soluzione IT-Wallet. 
```

peppelinux commented on docs/it/functionalities.rst:
```diff
@@ -0,0 +1,543 @@
+Tale presentazione può avvenire in due diverse modalità, sulla base del tipo di dispositivo utilizzato per accedere al servizio di interesse: 
+
+- **Stesso dispositivo**: nel caso in cui l’Utente voglia accedere a un servizio digitale online utilizzando lo stesso dispositivo su cui ha installato l’Istanza IT-Wallet; 
+- **Cross-dispositivo**: nel caso in cui l'Utente voglia accedere a un servizio digitale utilizzando un dispositivo differente rispetto a quello in cui ha installato l’Istanza IT-Wallet. 
```
```suggestion
- **Altro dispositivo**: nel caso in cui l'Utente voglia accedere a un servizio digitale utilizzando un dispositivo differente rispetto a quello in cui ha installato l’Istanza IT-Wallet. 
```

ChiaLeon replied:
Proponiamo di sostituire con versione inglese

```suggestion
- **Cross-device**: nel caso in cui l'Utente voglia accedere a un servizio digitale utilizzando un dispositivo differente rispetto a quello in cui ha installato l’Istanza IT-Wallet. 
```

peppelinux commented on docs/it/functionalities.rst:
```diff
@@ -0,0 +1,543 @@
+- **Stesso dispositivo**: nel caso in cui l’Utente voglia accedere a un servizio digitale online utilizzando lo stesso dispositivo su cui ha installato l’Istanza IT-Wallet; 
+- **Cross-dispositivo**: nel caso in cui l'Utente voglia accedere a un servizio digitale utilizzando un dispositivo differente rispetto a quello in cui ha installato l’Istanza IT-Wallet. 
+
+Di seguito i requisiti funzionali a supporto dell’Esperienza Utente relativi a entrambe le modalità che il Fornitore di Soluzione IT-Wallet DEVE garantire attraverso la propria Soluzione IT-Wallet. 
```
```suggestion
Di seguito i momenti caratterizzanti l’Esperienza Utente e relativi a entrambe le modalità. 
```

steps88 replied:
```suggestion
Di seguito i requisiti dell’Esperienza Utente relativi a entrambe le modalità che il Fornitore di Soluzione IT-Wallet DEVE garantire attraverso la propria Soluzione IT-Wallet. 
```

peppelinux commented on docs/it/functionalities.rst:
```diff
@@ -0,0 +1,543 @@
+
+In caso di errori nell’utilizzo dell’Istanza IT-Wallet, il Fornitore di Soluzione IT-Wallet DEVE garantire all’Utente la visualizzazione di messaggi coerenti che lo informino e guidino alla loro risoluzione. Per approfondimenti si rimanda alla sezione :ref:`functionalities:Gestione degli errori`. 
+
+**Cross-dispositivo** 
```
```suggestion
**Altro dispositivo** 
```

ChiaLeon replied:
Proponiamo di utilizzare la dicitura in inglese (vedi altri commenti)

```suggestion
**Cross-device** 
```

peppelinux commented on docs/it/functionalities.rst:
```diff
@@ -0,0 +1,543 @@
+Autenticazione 
+""""""""""""""
+
+L'Autenticazione è un caso d’uso specifico della presentazione remota e consente all’Utente di accedere in modo sicuro ai servizi resi disponibili da Verificatori di Attestati Elettronici pubblici e privati, presentando il PID ed eventualmente un set di Attributi contenuti negli Attestati Elettronici di Attributi ottenuti. Questo garantisce all’Utente il pieno controllo sui propri dati e la possi ...
```
```suggestion
L'Autenticazione è un caso d’uso specifico della presentazione remota e consente all’Utente di accedere in modo sicuro ai servizi resi disponibili da Verificatori di Attestati Elettronici pubblici e privati, presentando il PID ed eventualmente un set di Attributi contenuti negli Attestati Elettronici di Attributi ottenuti. Questo garantisce all’Utente il pieno controllo sui propri dati e la possibilità di condividere anche i soli dati strettamente necessari alla verifica da parte del Verificatore di Attestati Elettronici. 
```

peppelinux commented on docs/it/functionalities.rst:
```diff
@@ -0,0 +1,543 @@
+
+L'Autenticazione è un caso d’uso specifico della presentazione remota e consente all’Utente di accedere in modo sicuro ai servizi resi disponibili da Verificatori di Attestati Elettronici pubblici e privati, presentando il PID ed eventualmente un set di Attributi contenuti negli Attestati Elettronici di Attributi ottenuti. Questo garantisce all’Utente il pieno controllo sui propri dati e la possi ...
+
+Il processo di Autenticazione può avvenire in entrambe le modalità stesso dispositivo e cross-dispositivo descritte sopra. Per quanto riguarda i requisiti funzionali a supporto dell’Esperienza Utente, si DEVONO rispettare gli stessi requisiti previsti per la presentazione in remoto nelle due modalità (stesso dispositivo e cross-dispositivo). 
```
```suggestion
Il processo di Autenticazione può avvenire in entrambe le modalità Stesso dispositivo e Altro dispositivo descritte sopra. Per quanto riguarda i requisiti funzionali a supporto dell’Esperienza Utente, si DEVONO rispettare gli stessi requisiti previsti per la presentazione in remoto nelle due modalità (stesso dispositivo e cross-dispositivo). 
```

siamo sicuri di italianizzare Stesso dispositivo e Altro dispositivo?

io lascerei same e cross device

steps88 replied:
Concordiamo sull'usare la versione inglese. Ritieni utile aggiungerli al glossario?

pcerqua-ipzs replied:
```suggestion
Il processo di Autenticazione può avvenire in entrambe le modalità Stesso dispositivo e cross-device descritte sopra. Per quanto riguarda i requisiti funzionali a supporto dell’Esperienza Utente, si DEVONO rispettare gli stessi requisiti previsti per la presentazione in remoto nelle due modalità (stesso dispositivo e cross-dispositivo).
```

peppelinux commented on docs/it/functionalities.rst:
```diff
@@ -0,0 +1,543 @@
+
+Al fine di garantire un processo di Autenticazione adeguato e coerente tra tutti i Verificatori di Attestati Elettronici, ciascun Verificatore di Attestati Elettronici DEVE rispettare i requisiti relativi all’aspetto grafico e all’Esperienza Utente descritti di seguito e DOVREBBE utilizzare le Risorse Ufficiali open source. 
+
+Qualora i Verificatori di Attestati Elettronici non intendano utilizzare tali risorse open source, POSSONO sviluppare in autonomia le Soluzioni Tecniche abilitanti il flusso di Autenticazione ma DEVONO rispettare i requisiti sotto dettagliati, unitamente al rispetto di [RIF_ACCESSIBILITÀ] e, nel caso di enti pubblici, delle [LG_DESIGN].   
```
```suggestion
I Verificatori di Attestati Elettronici DEVONO rispettare i requisiti sotto dettagliati, unitamente al rispetto di [RIF_ACCESSIBILITÀ] e, nel caso di enti pubblici, delle [LG_DESIGN]. Qualora i Verificatori di Attestati Elettronici non intendano utilizzare tali risorse open source, questi POSSONO sviluppare in autonomia le Soluzioni Tecniche abilitanti il flusso di Autenticazione.   
```

peppelinux replied:
quando normiamo, usiamo sempre e chiaramente il soggetto al quale il linguaggio normativo si applica

steps88 replied:
Suggeriamo di integrare la modifica nel paragrafo precedente:

Al fine di garantire un processo di Autenticazione adeguato e coerente tra tutti i Verificatori di Attestati Elettronici, ciascun Verificatore di Attestati Elettronici DEVE rispettare i requisiti relativi all’aspetto grafico e all’Esperienza Utente descritti di seguito, **unitamente al rispetto di [RIF_ACCESSIBILITÀ] e, nel caso di enti pubblici, delle [LG_DESIGN].** 
I Verificatori di Attestati Elettronici DOVREBBERO utilizzare le Risorse Ufficiali open source. Qualora non intendano utilizzare tali risorse open source, questi POSSONO sviluppare in autonomia le Soluzioni Tecniche abilitanti il flusso di Autenticazione.

peppelinux commented on docs/it/functionalities.rst:
```diff
@@ -0,0 +1,543 @@
+I Verificatori di Attestati Elettronici, in ogni caso, DEVONO abilitare il processo di Autenticazione rendendo disponibili le seguenti pagine: 
+
+- **Discovery Page**: ha l’obiettivo di mostrare all’Utente tutti i metodi di Autenticazione disponibili; 
+- **QR code page** (*solo per modalità cross-dispositivo*): ha lo scopo di invitare l’Utente a inquadrare il codice QR;  
+- **waiting page** (*solo per modalità cross-dispositivo*): ha lo scopo di invitare l’Utente a continuare il processo di Autenticazione sulla propria Istanza IT-Wallet; 
```
rivediamo questi termini cross dispositivo e stesso dispositivo

ChiaLeon replied:
```suggestion
- **QR code page** (*solo per modalità cross-device*): ha lo scopo di invitare l’Utente a inquadrare il codice QR;  
- **waiting page** (*solo per modalità cross-device*): ha lo scopo di invitare l’Utente a continuare il processo di Autenticazione sulla propria Istanza IT-Wallet; 
```

peppelinux commented on docs/it/functionalities.rst:
```diff
@@ -0,0 +1,543 @@
+- **thank you page**: ha lo scopo di comunicare all’Utente l'avvenuta Autenticazione; 
+- **error page**: ha lo scopo di comunicare all’Utente eventuali errori legati al flusso di Autenticazione. 
+
+Tali pagine DEVONO prevedere elementi trasversali ricorrenti, in continuità con l’Identità Visiva del Touchpoint del Verificatore di Attestati Elettronici, in particolare: 
```
se usiamo DEVONO dobbiamo essere chiari su cosa, come i seguenti. Non possiamo usare DEVONO se non siamo chiari su cosa DEVONO

ChiaLeon replied:
```suggestion
Tali pagine DEVONO prevedere i seguenti elementi trasversali ricorrenti, in continuità con l’Identità Visiva del Touchpoint del Verificatore di Attestati Elettronici:
```

peppelinux commented on docs/it/functionalities.rst:
```diff
@@ -0,0 +1,543 @@
+
+In ogni caso: 
+
+- la pagina DEVE presentare tutte le modalità di Autenticazione attraverso l’identità digitale tra cui la modalità di Autenticazione del Sistema IT-Wallet, quindi tramite il Pulsante di Autenticazione; 
+- la pagina PUÒ presentare anche modalità di Autenticazione alternative, se disponibili; 
+- la pagina DOVREBBE garantire informazioni minime a supporto, per permettere all’Utente di compiere una scelta consapevole e informata. 
```
il linguaggio normativo non si applica alla pagina ma a chi la implementa, in questo caso il Verificatore di Attestati Elettronici DEVE implementare la pagina ...

ChiaLeon replied:
```suggestion
In ogni caso, nella pagina, il Verificatore di Attestati Elettronici:
- DEVE garantire la presenza di tutte le modalità di Autenticazione attraverso l’identità digitale tra cui la modalità di Autenticazione del Sistema IT-Wallet, quindi tramite il Pulsante di Autenticazione; 
- PUÒ presentare anche modalità di Autenticazione alternative, se disponibili; 
- DOVREBBE garantire informazioni minime a supporto, per permettere all’Utente di compiere una scelta consapevole e informata. 
```

Da modificare anche negli altri punti elenco di questa sezione (Autenticazione)

peppelinux commented on docs/it/functionalities.rst:
```diff
@@ -0,0 +1,543 @@
+- la pagina PUÒ presentare anche modalità di Autenticazione alternative, se disponibili; 
+- la pagina DOVREBBE garantire informazioni minime a supporto, per permettere all’Utente di compiere una scelta consapevole e informata. 
+
+Nel caso l’Utente stia navigando la pagina del Verificatore di Attestati Elettronici da un Touchpoint diverso da quello su cui ha attivato l’Istanza IT-Wallet (modalità cross-dispositivo), la scelta di Autenticazione tramite il Sistema IT-Wallet DEVE condurre l’Utente alla QR code page. 
```
rivediamo i termini quali cross-dispositivo

ChiaLeon replied:
```suggestion
Nel caso l’Utente stia navigando la pagina del Verificatore di Attestati Elettronici da un Touchpoint diverso da quello su cui ha attivato l’Istanza IT-Wallet (modalità cross-device), la scelta di Autenticazione tramite il Sistema IT-Wallet DEVE condurre l’Utente alla QR code page. 
```

peppelinux commented on docs/it/functionalities.rst:
```diff
@@ -0,0 +1,543 @@
+
+Nel caso l’Utente stia navigando la pagina del Verificatore di Attestati Elettronici da un Touchpoint diverso da quello su cui ha attivato l’Istanza IT-Wallet (modalità cross-dispositivo), la scelta di Autenticazione tramite il Sistema IT-Wallet DEVE condurre l’Utente alla QR code page. 
+
+Nel caso in cui invece l’Utente stia navigando la pagina del Verificatore di Attestati Elettronici dallo stesso Touchpoint su cui ha attivato l’Istanza IT-Wallet (modalità stesso dispositivo) tale pagina DEVE condurre l’Utente all’apertura della propria Istanza IT-Wallet. 
```
rivediamo stesso dispositivo

ChiaLeon replied:
```suggestion
Nel caso in cui invece l’Utente stia navigando la pagina del Verificatore di Attestati Elettronici dallo stesso Touchpoint su cui ha attivato l’Istanza IT-Wallet (modalità same-device) tale pagina DEVE condurre l’Utente all’apertura della propria Istanza IT-Wallet. 
```

peppelinux commented on docs/it/functionalities.rst:
```diff
@@ -0,0 +1,543 @@
+
+**QR code page (*solo per modalità cross-dispositivo*)** 
+
+La QR code page è la pagina su cui atterra l’Utente che ha scelto l’Autenticazione tramite il Sistema IT-Wallet in un flusso cross-dispositivo, e ha lo scopo di invitare l’Utente a scannerizzare, con la propria Istanza IT-Wallet, il codice QR generato. 
```
rivediamo cross dispositivo

ChiaLeon replied:
```suggestion
La QR code page è la pagina su cui atterra l’Utente che ha scelto l’Autenticazione tramite il Sistema IT-Wallet in un flusso cross-device, e ha lo scopo di invitare l’Utente a scannerizzare, con la propria Istanza IT-Wallet, il codice QR generato. 
```

peppelinux commented on docs/it/functionalities.rst:
```diff
@@ -0,0 +1,543 @@
+Per garantire la leggibilità del codice QR: 
+
+- si DEVENO rispettare le dimensioni minime raccomandate per garantire una scansione efficace. Una misura di 150x150 pixel è generalmente adeguata, ma per codici con alta densità di dati (e.g. URL lunghi o numerosi caratteri), è consigliabile aumentarla a 300x300 pixel o più; 
+- si DEVE mantenere un contrasto adeguato tra il codice QR e lo sfondo (la condizione ideale prevede uno sfondo bianco con un codice QR nero); 
```
qui bisogna normare i colori, il contrasto di conseguenza è sempre meglio bianco/nero

non lascerei la libertà agli implementatori di supporre soluzioni di contrasto presumibilmente adeguate

ChiaLeon replied:
```suggestion
Inoltre, nei rispetto di [RIF_ACCESSIBILITÁ], relativamente al codice QR, il Verificatore di Attestati Elettronici: 

- DEVE rispettare le dimensioni minime raccomandate per garantire una scansione efficace. Una misura di 150x150 pixel è generalmente adeguata, ma per codici con alta densità di dati (e.g. URL lunghi o numerosi caratteri), è consigliabile aumentarla a 300x300 pixel o più; 
- DEVE garantire un contrasto minimo tra il codice QR e lo sfondo (la condizione ideale prevede uno sfondo bianco con un codice QR nero); 
- DEVE evitare inversioni di colore tra sfondo e codice QR; 
- DEVE limitare la presenza a un solo codice QR per pagina; 
- DEVE garantire nitidezza e alta qualità; 
- DEVE garantire il formato SVG; 
- DEVE garantire che non venga parzialmente nascosto da testo o altri elementi.
```

peppelinux commented on docs/it/functionalities.rst:
```diff
@@ -0,0 +1,543 @@
+- si DEVE mantenere un contrasto adeguato tra il codice QR e lo sfondo (la condizione ideale prevede uno sfondo bianco con un codice QR nero); 
+- si DEVONO evitare inversioni di colore tra sfondo e codice QR; 
+- si DEVE limitare la presenza a un solo codice QR per pagina; 
+- il codice QR DEVE essere nitido e di alta qualità (si consiglia il formato SVG); 
```
```suggestion
- il codice QR DEVE essere nitido e di alta qualità;
- il codice QR DEVE essere in formato SVG; 
```

ChiaLeon replied:
risolto (vedi commento sopra)

peppelinux commented on docs/it/functionalities.rst:
```diff
@@ -0,0 +1,543 @@
+
+La waiting page è la pagina che invita l’Utente a proseguire il processo di Autenticazione sulla propria Istanza IT-Wallet, a valle della scansione del codice QR. 
+
+Il Verificatore di Attestati Elettronici DOVREBBE implementare la waiting page (cross-dispositivo) resa disponibile nelle Risorse Ufficiali. In ogni caso: 
```
rivediamo cross dispositivo

ChiaLeon replied:
```suggestion
Il Verificatore di Attestati Elettronici DOVREBBE implementare la waiting page (cross-device) resa disponibile nelle Risorse Ufficiali. In ogni caso: 
```

peppelinux commented on docs/it/functionalities.rst:
```diff
@@ -0,0 +1,543 @@
+
+In questa sezione sono illustrate tre diverse categorie di requisiti per la gestione del ciclo di vita di ogni Attestato Elettronico e nello specifico per la gestione: 
+
+- **del suo stato**: per consentire all’Utente di appurare la condizione di validità o invalidità di un Attestato Elettronico; 
+- **dei suoi utilizzi**: per consentire all’Utente di visualizzare e gestire lo storico delle transazioni effettuate con un Attestato Elettronico; 
+- **dei suoi dati**: per consentire all’Utente di archiviare e ripristinare ogni Attestato Elettronico di Attributi in linea col principio di *data portability*. 
+
+Di seguito i principali aspetti che impattano e determinano l’Esperienza Utente nell’ambito della gestione degli Attestati Elettronici per mezzo di un’Istanza IT-Wallet e i requisiti funzionali riferiti a ciascuna categoria. 
```
qui bisogna aggiungere delle considerazioni di privacy

bisogna chiarire dove materialmente risiedano i dati dell'utente

ChiaLeon replied:
se esiste già un contenuto di approfondimento, possiamo aggiungere un rimando

peppelinux commented on docs/it/functionalities.rst:
```diff
@@ -0,0 +1,543 @@
+   - non sussistono più i requisiti minimi di sicurezza e/o affidabilità di una o più delle parti coinvolte; 
+   - il dispositivo dell'Utente non soddisfa più i requisiti minimi di sicurezza (dispositivo rootato o jailbroken). 
+ 
+Storico degli Attestati Elettronici 
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Al fine di garantire i principi di visibilità e trasparenza, il Fornitore di Soluzione IT-Wallet DEVE permettere all’Utente di visualizzare lo storico di tutte le transazioni effettuate, ovvero l’utilizzo degli Attestati Elettronici di Attributi tramite l'Istanza IT-Wallet. In particolare: 
+
+- l’Istanza IT-Wallet DEVE mostrare all’Utente con quali Verificatori di Attestati Elettronici ha interagito e quali Attestati Elettronici sono stati oggetto di presentazione e verifica; 
+- l’Istanza IT-Wallet DEVE permettere all’Utente di richiedere facilmente al Verificatore di Attestati Elettronici la cancellazione delle proprie informazioni oggetto delle precedenti presentazioni. 
```
non è in sovrapposizione con una sezione precedente dove ho commentato sulle considerazioni di privacy?

ChiaLeon replied:
Nella sezione precedente vengono introdotte le successive sezioni di approfondimento, tra cui questa relativa allo Storico (al momento abbozzate in attesa di approfondimenti progettuali).

peppelinux commented on docs/it/functionalities.rst:
```diff
@@ -0,0 +1,543 @@
+
+Una comunicazione efficace in caso di errore determina un vantaggio anche per gli attori coinvolti, in quanto concorre alla riduzione delle richieste di assistenza e, quindi, alla minimizzazione dell’impatto sui sistemi.  
+
+Di seguito vengono presentati i requisiti e le principali buone pratiche di gestione dell’errore, riferite alle modalità di interazione tra l’Utente e la propria Istanza IT-Wallet. Tutti gli Attori Primari coinvolti nell'erogazione del servizio DEVONO inviare all’Utente, tramite l’Istanza IT-Wallet, input puntuali e granulari, ciascuno per i propri ambiti di competenza. Ciascun attore coinvolto D ...
```
non possiamo usare DEVE se non siamo precisi nell'indicare cosa di deve implementare

consiglio di rimandare nelle sezioni tecniche dove gli errori vengono definiti

pcerqua-ipzs replied:
#729

peppelinux commented on docs/it/functionalities.rst:
```diff
@@ -0,0 +1,543 @@
+
+- **Comunicazione efficiente**: garantire all’Utente la possibilità di monitorare lo stato aggiornato della propria richiesta durante tutte le fasi di lavorazione, attraverso una comunicazione chiara, continua e coordinata.  
+
+Per applicare queste buone pratiche, gli attori coinvolti DOVREBBERO attuare i seguenti requisiti in modo da strutturare un modello di assistenza a livelli gerarchici: 
```
```suggestion
Per applicare queste buone pratiche, gli attori coinvolti implementano i seguenti livelli di assistenza gerarchici: 
```

RosaliaGaleano replied:
Non eliminerei il linguaggio normativo perché penso sia importante spingere gli attori a seguire il modello proposto. Potremmo riformulare così.
```suggestion
Per applicare queste buone pratiche, gli attori coinvolti DOVREBBERO implementare i seguenti livelli di assistenza gerarchici: 
```

peppelinux commented on docs/it/functionalities.rst:
```diff
@@ -0,0 +1,543 @@
+
+	1. **I Livello – Gestione autonoma**: il Fornitore di Soluzione IT-Wallet DOVREBBE permettere all’Utente di disporre di una sezione di domande frequenti (FAQ) all’interno della propria Istanza IT-Wallet per chiarire dubbi e risolvere in autonomia alcune problematiche. Ogni attore DOVREBBE formulare delle domande frequenti e relative risposte specifiche rispetto a dati e funzionalità messe a disp ...
+
+	2. **II Livello – Richiesta di assistenza al Fornitore di Soluzione IT-Wallet**: se il I livello non fosse sufficiente, il Fornitore di Soluzione IT-Wallet DEVE permettere all’Utente di aprire una o più richieste di assistenza.  Tali richieste DOVREBBERO essere gestite tramite l’Istanza IT-Wallet o altri Touchpoint del Fornitore di Soluzione IT-Wallet. Il Fornitore di Soluzione IT-Wallet DEVE ef ...
```
questo deve è possibile solo se il livello 1 fosse implementato

tuttavia il livello 1 non normato come obbligatorio

fixare

RosaliaGaleano replied:
sì, è corretto. Inserirei DOVREBBERO anche al punto 2

RosaliaGaleano replied:
Suggerisco questa riformulazione:

```suggestion
	2. **II Livello – Richiesta di assistenza al Fornitore di Soluzione IT-Wallet**: se il I livello non fosse sufficiente, il Fornitore di Soluzione IT-Wallet DOVREBBE permettere all’Utente di aprire una o più richieste di assistenza, effettuare una diagnosi e procedere alla risoluzione della problematica, se di sua competenza. Tali richieste DOVREBBERO essere gestite tramite l’Istanza IT-Wallet o altri Touchpoint del Fornitore di Soluzione IT-Wallet.
```

peppelinux commented on docs/it/functionalities.rst:
```diff
@@ -0,0 +1,543 @@
+
+	2. **II Livello – Richiesta di assistenza al Fornitore di Soluzione IT-Wallet**: se il I livello non fosse sufficiente, il Fornitore di Soluzione IT-Wallet DEVE permettere all’Utente di aprire una o più richieste di assistenza.  Tali richieste DOVREBBERO essere gestite tramite l’Istanza IT-Wallet o altri Touchpoint del Fornitore di Soluzione IT-Wallet. Il Fornitore di Soluzione IT-Wallet DEVE ef ...
+
+	3. **III Livello – Inoltro della richiesta all’attore responsabile della problematica**: se il II livello non fosse sufficiente, il Fornitore di Soluzione IT-Wallet DOVREBBE garantire che la richiesta sia inoltrata all’attore responsabile (Fornitore di Attestati Elettronici di Attributi, Fornitore di Dati di Identificazione Personale e Titolare di Fonte Autentica) che DEVE farsi carico della ris ...
```
```suggestion
	3. **III Livello – Inoltro della richiesta all’attore responsabile della problematica**: se il II livello non fosse sufficiente, il Fornitore di Soluzione IT-Wallet DOVREBBE garantire che la richiesta sia inoltrata all’attore responsabile (Fornitore di Attestati Elettronici di Attributi, Fornitore di Dati di Identificazione Personale e Titolare di Fonte Autentica) che si fa carico della risoluzione della problematica e comunicare l’esito all’Utente. 
```

peppelinux commented on docs/it/functionalities.rst:
```diff
@@ -0,0 +1,543 @@
+
+	3. **III Livello – Inoltro della richiesta all’attore responsabile della problematica**: se il II livello non fosse sufficiente, il Fornitore di Soluzione IT-Wallet DOVREBBE garantire che la richiesta sia inoltrata all’attore responsabile (Fornitore di Attestati Elettronici di Attributi, Fornitore di Dati di Identificazione Personale e Titolare di Fonte Autentica) che DEVE farsi carico della ris ...
+
+Di seguito i requisiti funzionali a supporto dell’Esperienza Utente relativi all’Assistenza che il Fornitore di Soluzione IT-Wallet DEVE garantire attraverso la propria Soluzione IT-Wallet: 
```
anche questi, non sono requisiti funzionali

ridefinire lo scopo dei seguenti punti, rifrasando questo testo introduttivo

RosaliaGaleano replied:
In coerenza con i nostri commenti alle sezioni precedenti suggerirei la seguente riformulazione:

```suggestion
Di seguito i requisiti di Esperienza Utente che il Fornitore di Soluzione IT-Wallet DEVE garantire attraverso la propria Soluzione IT-Wallet: 
```

pcerqua-ipzs committed:
Merge remote-tracking branch 'pcerqua-ipzs/versione-corrente' into italian-translation-setup

# Conflicts:
#	docs/en/functionalities.rst

pcerqua-ipzs committed:
fix: removed explicit rst label

pcerqua-ipzs committed:
fix: template title

pcerqua-ipzs committed:
fix: aligned functionalities translations, spacings and title levels

pcerqua-ipzs committed:
Update docs/it/brand-identity.rst

Co-authored-by: Giuseppe De Marco <demarcog83@gmail.com>

pcerqua-ipzs committed:
chore: conf.py enhancements

pcerqua-ipzs committed:
chore: italian conf.py fixes

pcerqua-ipzs committed:
fix: aligned to https://github.com/italia/eid-wallet-it-docs/pull/708#discussion_r2104192687

pcerqua-ipzs committed:
Update docs/it/brand-identity.rst

fix: italian translation (english aligned in 67cf74abae29325674359558c3a016c2d670a81b)

Co-authored-by: steps88 <91465481+steps88@users.noreply.github.com>

pcerqua-ipzs committed:
Merge branch 'italia:italian-translation-setup-italia' into 'pcerqua-ipzs:italian-translation-setup'

pcerqua-ipzs committed:
fix: aligned to https://github.com/italia/eid-wallet-it-docs/pull/708#discussion_r2104192687

pcerqua-ipzs committed:
fix: italian translation (english aligned in 23a2d19064c0a64fb2515a7fc4f6ca11c004e203)

Co-authored-by: steps88 <91465481+steps88@users.noreply.github.com>

pcerqua-ipzs committed:
fix: aligned to https://github.com/italia/eid-wallet-it-docs/pull/708#discussion_r2104229095

pcerqua-ipzs committed:
fix: aligned to https://github.com/italia/eid-wallet-it-docs/pull/708#discussion_r2104229095

pcerqua-ipzs committed:
fix: italian translation (enhglish here: 92b5e16ad2a092a6ed011c752f67e8d71360cae2)

Co-authored-by: steps88 <91465481+steps88@users.noreply.github.com>

pcerqua-ipzs committed:
fix: aligned to https://github.com/italia/eid-wallet-it-docs/pull/708#discussion_r2104285370

pcerqua-ipzs committed:
fix: italian translation (english aligned in c619b9a38182e8b9eddb808cd0a95c93508ccc37)

Co-authored-by: steps88 <91465481+steps88@users.noreply.github.com>

pcerqua-ipzs committed:
fix: aligned to https://github.com/italia/eid-wallet-it-docs/pull/708#discussion_r2104285877

pcerqua-ipzs committed:
fix: italian translation (english aligned in 073410fbb373afc6b37e65ab5bf63bb91bd82299)

Co-authored-by: steps88 <91465481+steps88@users.noreply.github.com>

pcerqua-ipzs committed:
Update docs/it/brand-identity.rst

Co-authored-by: Giuseppe De Marco <demarcog83@gmail.com>

pcerqua-ipzs committed:
fix: aligned to https://github.com/italia/eid-wallet-it-docs/pull/708#discussion_r2104267821

pcerqua-ipzs committed:
fix: italian translation (english aligned in f8ced1b88d2ef94df6922f26342c8f6f10c0cf9c)

Co-authored-by: steps88 <91465481+steps88@users.noreply.github.com>

pcerqua-ipzs committed:
Update docs/it/functionalities.rst

Co-authored-by: Chiara Leonardi <112159908+ChiaLeon@users.noreply.github.com>

pcerqua-ipzs committed:
Update docs/it/functionalities.rst

Co-authored-by: Chiara Leonardi <112159908+ChiaLeon@users.noreply.github.com>

pcerqua-ipzs committed:
Update docs/it/functionalities.rst

Co-authored-by: Chiara Leonardi <112159908+ChiaLeon@users.noreply.github.com>

pcerqua-ipzs committed:
Update docs/it/functionalities.rst

Co-authored-by: Chiara Leonardi <112159908+ChiaLeon@users.noreply.github.com>

pcerqua-ipzs committed:
fix: aligned to https://github.com/italia/eid-wallet-it-docs/pull/708#discussion_r2104287123

pcerqua-ipzs committed:
fix: italian translation (english aligned in 29f56d450e437385d762ce4f80068c3fcdca8720)

Co-authored-by: steps88 <91465481+steps88@users.noreply.github.com>

pcerqua-ipzs committed:
fix: aligned to https://github.com/italia/eid-wallet-it-docs/pull/708#discussion_r2096402347

pcerqua-ipzs committed:
fix: italian translation (english aligned in 15a52a209d56d03e05c709712ebb54cb0a33a610)

Co-authored-by: Giuseppe De Marco <demarcog83@gmail.com>

pcerqua-ipzs committed:
Update docs/it/functionalities.rst

pcerqua-ipzs committed:
fix: aligned to https://github.com/italia/eid-wallet-it-docs/pull/708#discussion_r2096422720

pcerqua-ipzs committed:
fix: alignment

pcerqua-ipzs committed:
Update docs/it/functionalities.rst

Co-authored-by: Chiara Leonardi <112159908+ChiaLeon@users.noreply.github.com>

pcerqua-ipzs committed:
fix: italian translation (english aligned in 7183357c33572d84315b5dee44bd7cc1dd2a0e95)

Co-authored-by: Giuseppe De Marco <demarcog83@gmail.com>

pcerqua-ipzs committed:
Update docs/it/functionalities.rst

Co-authored-by: Chiara Leonardi <112159908+ChiaLeon@users.noreply.github.com>

pcerqua-ipzs committed:
fix: aligned with https://github.com/italia/eid-wallet-it-docs/pull/708#discussion_r2104817080

pcerqua-ipzs committed:
fix: italian translation (english aligned in 9be3a24eff9625e5162682387aade2df9ffd4b4a)

Co-authored-by: Lia <139998796+RosaliaGaleano@users.noreply.github.com>

pcerqua-ipzs committed:
fix: aligne to https://github.com/italia/eid-wallet-it-docs/pull/708#discussion_r2104822164 and https://github.com/italia/eid-wallet-it-docs/pull/708#discussion_r2104812723

pcerqua-ipzs committed:
Update docs/it/functionalities.rst

Co-authored-by: Giuseppe De Marco <demarcog83@gmail.com>

pcerqua-ipzs committed:
Update docs/it/functionalities.rst

Co-authored-by: Chiara Leonardi <112159908+ChiaLeon@users.noreply.github.com>

pcerqua-ipzs committed:
fix: italian translation (english aligned in 9f21e2dbae52906772c93a4566ef293e921c1d5c)

Co-authored-by: Lia <139998796+RosaliaGaleano@users.noreply.github.com>

pcerqua-ipzs committed:
fix: italian translation (english aligned in 9f21e2dbae52906772c93a4566ef293e921c1d5c)

Co-authored-by: Lia <139998796+RosaliaGaleano@users.noreply.github.com>

peppelinux commented on docs/it/functionalities.rst:
```diff
@@ -0,0 +1,543 @@
+
+A seconda delle specifiche esigenze dell’Utente, della tipologia di Attestato Elettronico di Attributi e delle disponibilità offerte dal Fornitore di Soluzione IT-Wallet, dal Fornitore di Attestati Elettronici di Attributi e dal Titolare di Fonte Autentica, l’ottenimento degli Attestati Elettronici di Attributi può avvenire attraverso due modalità: 
+
+- **dal Catalogo dell’Istanza IT-Wallet**: l’Utente esplora l’elenco degli Attestati Elettronici di Attributi messi a disposizione dalla Soluzione IT-Wallet, seleziona quello di suo interesse e avvia la procedura di richiesta che si conclude con l’ottenimento dell’Attestato Elettronico di Attributi nell’Istanza IT-Wallet; 
```
```suggestion
- **dal Catalogo dell’Istanza IT-Wallet**: l’Utente esplora l’elenco degli Attestati Elettronici di Attributi messi a disposizione dalla Soluzione IT-Wallet, seleziona quello di suo interesse e avvia la procedura di richiesta che si conclude con l’ottenimento dell’Attestato Elettronico di Attributi nell’Istanza IT-Wallet (vedi :ref:`Digital Credentials Catalogue <registry-catalogue.rst>`); 
```

peppelinux commented on docs/it/functionalities.rst:
```diff
@@ -0,0 +1,543 @@
+
+Al fine di garantire un processo di Autenticazione adeguato e coerente tra tutti i Verificatori di Attestati Elettronici, ciascun Verificatore di Attestati Elettronici DEVE rispettare i requisiti relativi all’aspetto grafico e all’Esperienza Utente descritti di seguito e DOVREBBE utilizzare le Risorse Ufficiali open source. 
+
+Qualora i Verificatori di Attestati Elettronici non intendano utilizzare tali risorse open source, POSSONO sviluppare in autonomia le Soluzioni Tecniche abilitanti il flusso di Autenticazione ma DEVONO rispettare i requisiti sotto dettagliati, unitamente al rispetto di [RIF_ACCESSIBILITÀ] e, nel caso di enti pubblici, delle [LG_DESIGN].   
```
```suggestion
Al fine di garantire un processo di Autenticazione adeguato e coerente tra tutti i Verificatori di Attestati Elettronici, ciascun Verificatore di Attestati Elettronici DEVE rispettare i requisiti relativi all’aspetto grafico e all’Esperienza Utente descritti di seguito, unitamente al rispetto di [RIF_ACCESSIBILITÀ] e, nel caso di enti pubblici, delle [LG_DESIGN].

I Verificatori di Attestati Elettronici DOVREBBERO utilizzare le Risorse Ufficiali open source. Qualora non intendano utilizzare tali risorse open source, i Verificatori di Attestati Elettronici POSSONO sviluppare in autonomia le Soluzioni Tecniche abilitanti il flusso di Autenticazione.
```

peppelinux commented on docs/it/functionalities.rst:
```diff
@@ -0,0 +1,543 @@
+In questa sezione sono illustrate tre diverse categorie di requisiti per la gestione del ciclo di vita di ogni Attestato Elettronico e nello specifico per la gestione: 
+
+- **del suo stato**: per consentire all’Utente di appurare la condizione di validità o invalidità di un Attestato Elettronico; 
+- **dei suoi utilizzi**: per consentire all’Utente di visualizzare e gestire lo storico delle transazioni effettuate con un Attestato Elettronico; 
```
```suggestion
- **dei suoi utilizzi**: per consentire all’Utente di visualizzare e gestire lo storico delle transazioni effettuate utilizzando un Attestato Elettronico; 
```

peppelinux commented on docs/it/functionalities.rst:
```diff
@@ -0,0 +1,543 @@
+
+Una comunicazione efficace in caso di errore determina un vantaggio anche per gli attori coinvolti, in quanto concorre alla riduzione delle richieste di assistenza e, quindi, alla minimizzazione dell’impatto sui sistemi.  
+
+Di seguito vengono presentati i requisiti e le principali buone pratiche di gestione dell’errore, riferite alle modalità di interazione tra l’Utente e la propria Istanza IT-Wallet. Tutti gli Attori Primari coinvolti nell'erogazione del servizio DEVONO inviare all’Utente, tramite l’Istanza IT-Wallet, input puntuali e granulari, ciascuno per i propri ambiti di competenza. Ciascun attore coinvolto D ...
```
```suggestion
Di seguito vengono presentati i requisiti e le principali buone pratiche di gestione dell’errore, riferite alle modalità di interazione tra l’Utente e la propria Istanza IT-Wallet. Ciascun Attore Primario deve implementare una corretta gestione degli errori, in conformità alle attuali regole tecniche, al fine di comunicarli, direttamente o indirettamente, all’Utente e tramite l’Istanza IT-Wallet. Gli errori possono essere declinati, sulla base della loro natura, come segue: 
```

peppelinux committed:
fix: design, subject of the normative language

pcerqua-ipzs committed:
fix: alignements to https://github.com/italia/eid-wallet-it-docs/pull/708#discussion_r2107184792

pcerqua-ipzs committed:
fix: italian translation (english aligned in bca6a3adb77af61ae52078342ae5f7ff2ec6cea5)

Co-authored-by: Giuseppe De Marco <demarcog83@gmail.com>

pcerqua-ipzs committed:
Update docs/it/functionalities.rst

Co-authored-by: Giuseppe De Marco <demarcog83@gmail.com>

pcerqua-ipzs committed:
Update docs/it/functionalities.rst

Co-authored-by: Chiara Leonardi <112159908+ChiaLeon@users.noreply.github.com>

pcerqua-ipzs committed:
fix: aligned to https://github.com/italia/eid-wallet-it-docs/pull/708/files#r2107215943

pcerqua-ipzs committed:
fix: alignements

pcerqua-ipzs committed:
Update docs/it/functionalities.rst

Co-authored-by: Chiara Leonardi <112159908+ChiaLeon@users.noreply.github.com>

pcerqua-ipzs committed:
fix: removed doplications

pcerqua-ipzs committed:
fix: alignement to https://github.com/italia/eid-wallet-it-docs/pull/708#discussion_r2107215943

pcerqua-ipzs committed:
fix: italian translation (english aligned in 1b60e0a2b06cd35f600a3aee6bd573f1606372f2)

Co-authored-by: Giuseppe De Marco <demarcog83@gmail.com>

pcerqua-ipzs committed:
fix: typo fix

pcerqua-ipzs committed:
Update docs/it/functionalities.rst

Co-authored-by: Giuseppe De Marco <demarcog83@gmail.com>

pcerqua-ipzs committed:
fix: alignement to: https://github.com/italia/eid-wallet-it-docs/pull/708#discussion_r2107251441

pcerqua-ipzs committed:
fix: alignments between en and it

pcerqua-ipzs committed:
fix: italian translation (english aligned in e75f1c0fd59319b9004a578e2fdd9d0bce045f48)

Co-authored-by: Giuseppe De Marco <demarcog83@gmail.com>

pcerqua-ipzs committed:
fix: aligned to https://github.com/italia/eid-wallet-it-docs/pull/708#discussion_r2104321393

pcerqua-ipzs committed:
fix: italian translation (english aligned in 8a1155471c9202074162d92a4f2f094d09335f9f)

Co-authored-by: Chiara Leonardi <112159908+ChiaLeon@users.noreply.github.com>

pcerqua-ipzs committed:
fix: alignment to https://github.com/italia/eid-wallet-it-docs/pull/708#discussion_r2107222125

pcerqua-ipzs committed:
fix: italian translation (english aligned in d21c3ff308459432e7fdaefeb8fe54abd5f0dc5c)

Co-authored-by: Giuseppe De Marco <demarcog83@gmail.com>

pcerqua-ipzs committed:
Merge remote-tracking branch 'pcerqua-ipzs/versione-corrente' into italian-translation-setup

# Conflicts:
#	docs/en/brand-identity.rst
#	docs/en/conf.py
#	docs/en/functionalities.rst

pcerqua-ipzs committed:
fix: alignement cleanup

pcerqua-ipzs committed:
fix: single quotes fix

pcerqua-ipzs committed:
Merge remote-tracking branch 'pcerqua-ipzs/versione-corrente' into italian-translation-setup

# Conflicts:
#	docs/it/defined-terms.rst

pcerqua-ipzs committed:
fix: toctree adaptations

pcerqua-ipzs committed:
fix: toctree update

grausof commented on utils/svg2pdf.py:
Should this empty file remain?

pcerqua-ipzs replied:
Yes, this file should remain. I don't know why seems empty, but contains a Python script.

pcerqua-ipzs replied:
in the last commit I have pushed also some blank spaces to check everything it's fine

pcerqua-ipzs committed:
fix: editorial fixes

pcerqua-ipzs committed:
fix: editorial change

pcerqua-ipzs commented on docs/it/defined-terms-and-references.rst:
```diff
@@ -0,0 +1,13 @@
+.. include:: ../common/common_definitions.rst
+
+
+Defined Terms and References
+============================
```
Proper translations for content that is not about Design Principles will be provided in future PRs.

pcerqua-ipzs commented on docs/it/architecture-overview.rst:
```diff
@@ -0,0 +1,36 @@
+.. include:: ../common/common_definitions.rst
+
+
+Architecture Overview
+=====================
```
Proper translations for content that is not about Design Principles will be provided in future PRs.

pcerqua-ipzs committed:
fix: new lines at end of files

pcerqua-ipzs committed:
fix: new lines
